plugin/package.json

```diff
@@ -27,20 +27,19 @@
     "pre-commit": "yarn run pretty-quick --staged --no-restage --bail --pattern \"**/*.{ts,tsx,scss,json}\" && npm run lint:precommit"
   },
   "dependencies": {
-    "@openshift-console/dynamic-plugin-sdk": "^0.0.19",
+    "@openshift-console/dynamic-plugin-sdk": "0.0.18",
     "@patternfly/patternfly": "4.224.2",
     "@patternfly/react-charts": "6.94.18",
     "@patternfly/react-core": "4.276.6",
     "@patternfly/react-table": "4.112.39",
-    "@patternfly/react-tokens": "4.94.6",
     "@patternfly/react-topology": "4.91.27",
-    "axios": "^0.21.4",
+    "axios": "^1.6.8",
     "bootstrap-slider-without-jquery": "10.0.0",
     "cy-node-html-label": "2.0.0",
     "cytoscape": "3.15.5",
     "cytoscape-canvas": "3.0.1",
     "cytoscape-popper": "1.0.7",
-    "dagre": "0.8.5",
+    "dagre": "0.8.2",
```

Comment:
`dagre` library is downgraded to match with `@patternfly/react-topology` dagre dependency and avoid library duplications